File: Java/1005.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        
        double A = leitor.nextDouble();
        double B = leitor.nextDouble();
        double MEDIA = ((A * 3.5) + (B * 7.5)) / 11;
        
        System.out.printf("MEDIA = %.5f\n", MEDIA);
    }
 
}

File: C/1005.c
#include <stdio.h>
int main() {
    double x, y, media;
    scanf("%lf", &x);
    scanf("%lf", &y);
    media = (x * 3.5 + y * 7.5) / (3.5 + 7.5);
    printf("MEDIA = %.5lf\n", media);
    return 0;
}